controls/srg_ctr/SRG-APP-000026-CTR-000070.yml

```diff
@@ -3,13 +3,7 @@ controls:
   levels:
   - medium
   title: {{{ full_name }}} must automatically audit account creation.
-  related_rules:
-  - idp_is_configured
-  - ocp_idp_no_htpasswd
-  - kubeadmin_removed
-  status: not applicable
-  status_justification: |-
-    Not Applicable. Applicable to Identity Management Provider and not
-    OCP. Only configurable check is to ensure OCP is configured for an
-    IDP under SRG-APP-000023-CTR-000055. Verify with IdM service provider
-    admins that the IdM meets the requirements.
+  status: automated
+  rules:
+  - audit_rules_sysadmin_actions
+  - audit_rules_usergroup_modification
```

Comment:
I believe this no newline here will fail at a test... you might want to quickly iterate over them

Reply:
yup, I was expecting CI to find small issues like these, but there's also:
```
Unable to find SRG Variables. Id in the control must be a valid SRGID.
```

Reply:
I believe it has something to do with the change in `controls/srg_ctr.yml`

I think you need to remove all the `controls` section` see above

Reply:
thanks, fixed